### ISSUE TITLE
API ?

### ISSUE THREAD

**Issue by uhq111** [javascript-obfuscator/javascript-obfuscator#1207]
Hello,

I'd like to know whether https://obfuscator.io/ has an API so that I can send my code directly and have the site send it back to me with an obfuscated version.

**Comment by slig** [comment]
No, the website runs 100% on the browser. You have to use the package
directly, via a bundler plugin, or create your own HTTP API and host it
yourself.

**Comment by uhq111** [comment]
Do you have a repo with an example?
